RFCs/009-break-data-driven-test-into-multiple-tests-objects.md

```diff
@@ -0,0 +1,25 @@
+#Break data driven test into multiple test objects
+
+##Summary
+Allow data-driven tests to start and stop recording between each execution.
+
+##Motivation 
+When recording videos of tests using Microsoft.VisualStudio.TestTools.DataCollection.VideoRecorder data-driven tests are treated as a single block causing every execution of the test to be within the same video. See Issue [450](https://github.com/Microsoft/testfx/issues/450).
+
+##Detailed Design
+
+### Requirements
+1. DataTestMethod Attribute should allow the option to treat it's tests as separate test objects.
+2. Should not alter current behavior for existing users.
+
+### Proposed solution
```

Comment:
There are multiple ways to data drive a test.
- using DataRow
- using DynamicData
- directly extending ITestDataSource

Is your proposal for all of these approaches?